package-lock.json

```diff
@@ -6640,6 +6640,12 @@
               }
             }
           }
+        },
```

Comment:
We didn't make any dep changes to the root (client release group's) package.json, so I don't think these changes should be here.

Reply:
Reverted `package-lock.json`